src/tokenizer/tokenizer.py

```diff
@@ -712,7 +712,13 @@ def gen_from_string(txt, replace_composite_glyphs=True, replace_html_escapes=Fal
     # newlines separated only by whitespace), we interpret
     # them as hard sentence boundaries
     first = True
-    for span in re.split(r"\n\s*\n", txt):
+    splitter = re.split(r"\n\s*\n", txt)
```

Comment:
Move this line down to an else clause on the if statement

Reply:
Done!